packages/ag-charts-enterprise/src/features/crosshair/crosshair.ts

```diff
@@ -196,12 +193,12 @@ export class Crosshair extends _ModuleSupport.BaseModuleInstance implements _Mod
         const { crosshairGroup, hoverRect } = this;
         const { offsetX, offsetY } = event;
 
-        if (event.type === 'hover' && this.visible && hoverRect.containsPoint(offsetX, offsetY)) {
-            crosshairGroup.visible = true;
-
+        if (hoverRect.containsPoint(offsetX, offsetY)) {
             const lineData = this.getData(event);
 
             this.updatePositions(lineData);
+
+            crosshairGroup.visible = true;
```

Comment:
In `hideCrosshairs()` it also hides the labels, do they need to be made visible here too or is that handled elsewhere?

Reply:
so that's done in the updatePositions function